pt/esbocos/originais/git-ferramenta-metodologica-projetos-historia-1.md

```diff
@@ -304,33 +304,33 @@ No ramo main
 No commits yet
 
 Arquivos não monitorados:
```

Comment:
Tendo em conta o glossário, "Arquivos" poderá ser alterado para "Ficheiros"? Ou no código aparece mesmo como "Arquivos"? Será pela utilização de PT do BR? Se for podemos deixar uma nota no início da lição.

Reply:
No próprio programa aparece "Arquivo", pois minha versão está em pt_BR.
Eu procurei uma versão traduzida para pt_PT mas não encontrei no repositório do Git. 
Aparentemente, não existe uma versãom específica do Git traduzida para pt de PT.

Reply:
Muito obrigada pela clarificação @ericbrasiln. Será que podemos colocar essa informação algures na lição? Referindo que a versão portuguesa é pt_BR. Por exemplo, na nota de rodapé 1, que já refere a existência da versão portuguesa. Poderia ficar algo como:

"A versão portuguesa do Git remete para o português do Brasil (pt_BR) não existindo, até ao momento, uma versão em português de Portugal (pt_PT). Esta  ainda apresenta trechos em inglês, o que pode gerar problemas de compreensão. Nesta lição utilizámos o padrão do Git existente em pt_BR."

O que acha?

Reply:
Perfeito!

Reply:
Inserido. Obrigada!